playwright/navigation.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { expect, test } from '@playwright/test';
+
+test('Can find the different components in the app', async ({ page }) => {
+  await page.goto('/');
+  await expect(page.getByTestId('homePageTitle')).toBeVisible();
+  await expect(page.getByTestId('homePageLink')).toBeVisible();
+  await expect(page.getByTestId('portfolioPageLink')).toBeVisible();
+  await expect(page.getByTestId('contactPageLink')).toBeVisible();
+
+  await Promise.all([
+    page.getByTestId('aboutPageLink').click(),
+    page.waitForNavigation(),
+  ]);
```

Comment:
as discussed, `page.waitForNavigation()` is deprecated and can be replaced with `page.waitForURL()`

```js
await page.click('a.delayed-navigation'); // Clicking the link will indirectly cause a navigation
await page.waitForURL('**/target.html');
```

https://playwright.dev/docs/api/class-page#page-wait-for-url

Reply:
Updated to use `page.waitForURL`
[10f606a](https://github.com/upleveled/next-js-portfolio-web/pull/107/commits/10f606a1ad3d180595e6886d7f020fca3cd484fd)